.envrc

```diff
@@ -0,0 +1 @@
+use flake .
```

Comment:
Hm yes, but I would argue this is also a general problem with direnv. You could also miss the required Python env. A compromise could be to name it differently, so that it isn't picked up by direnv and just symlink it, if it is to be used. But then again one could also simply not allow the direnv. Certainly, none is perfect.